libs/components/lists/src/assets/locales/resources_en_US.json

```
@@ -1,4 +1,8 @@
 {
+  "skyux_filter_aria_label_descriptor": {
+    "_description": "Default Default aria label for a filter button when a descriptor is given",
+    "message": "Filter {0}"
+  },
   "skyux_filter_button_title": {
     "_description": "Text for the filter button title",
```

```suggestion
    "_description": "Text for the filter button label",
```
I realize this is existing localization content that you aren't actually touching, and I'm not 100 percent sure on my suggestion. However, the naming seems off here. We don't refer to any "title" at all in the filter component docs, and the other properties here all use "label." I don't think we want to mess with the `skyux_filter_button_title` because I assume that would be a breaking change, but if I'm correct that our naming convention in this instance is wonky, maybe we should at least fix it in the description?

---

We actually use this string for both the text on the button and for the `title` attribute - so I'm inclined to leave this.

---

Works for me. I just thought it was asking because the terminology seemed off at a glance.